js/oop/Palindrome.js

```diff
@@ -21,7 +64,21 @@ document.getElementById("form").addEventListener("submit", function (event) {
     const word = event.target["word"].value;
     const method = event.target["method"].value;
 
-    const result = new Palindrome(word);
+    let isPalindrome;
+
+    if (method === "reverse") {
+      isPalindrome = new Palindrome(word).reverse();
+    } else if (method === "loop") {
+      isPalindrome = new Palindrome(word).loop();
+    } else if (method === "recursive") {
+      isPalindrome = new Palindrome(word).recursive();
+    } else {
+      throw new Error("Method must be reverse, loop, or recursive.");
+    }
+
+    const result = isPalindrome
+      ? "Yes, this word is a palindrome."
+      : "No, this word is not a palindrome.";
```

Comment:
This is not how OOP was created on purpose. Could be changed much better than this.

Reply:
Change it, but not sure if it correctly.

Reply:
Refactor this code by doing what I suggest on https://github.com/ngodingbang/rwid-git/pull/3#discussion_r1282726700.